frontend/app/components/ui/typography/index.ts

```
@@ -0,0 +1,16 @@
+export { Text } from './Text';
+export type { TextProps } from './Text';
+
+// Heading components (h1-h6)
+export { Heading } from './Heading';
+export type { HeadingProps } from './Heading';
+
+// Code block component
+// export { CodeBlock } from './CodeBlock';
+
+// Placeholder - components will be implemented in subsequent tasks
+export {}; 
```

Biome (1.9.4)

[error] This empty export is useless because there's another export or import.

This export makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)